app/src/main/java/org/sopt/at/screen/HomeScreen.kt

```diff
@@ -0,0 +1,217 @@
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shadow
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavHostController
+import org.sopt.at.R
+import org.sopt.at.data.TvProgram
+import org.sopt.at.viewmodel.AuthViewModel
+import org.sopt.at.viewmodel.HomeViewModel
+
+
+@Composable
+fun TopBar(navController: NavHostController, authViewModel: AuthViewModel) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.tving_logo),
+            contentDescription = "Tving Logo",
+            modifier = Modifier.size(80.dp)
+        )
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_tv),
+                contentDescription = "TV Icon",
+                tint = Color.White,
+                modifier = Modifier
+                    .size(50.dp)
+                    .padding(end = 16.dp)
+            )
+            Icon(
+                painter = painterResource(id = R.drawable.ic_profile),
+                contentDescription = "Profile Icon",
+                tint = Color.White,
+                modifier = Modifier
+                    .size(36.dp)
+                    .clickable {
+                        navController.navigate("my")
+                    })
+        }
+    }
+}
+
+
+@Composable
+fun BannerView(viewModel: HomeViewModel) {
+    LazyRow(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(480.dp),
+        horizontalArrangement = Arrangement.spacedBy(8.dp),
+        contentPadding = PaddingValues(horizontal = 16.dp)
+    ) {
+        items(viewModel.bannerList) { program ->
+            Image(
+                painter = painterResource(id = program.imageRes),
+                contentDescription = program.title,
+                contentScale = ContentScale.Crop,
+                modifier = Modifier
+                    .fillParentMaxWidth()
+                    .fillParentMaxHeight()
+                    .clip(RoundedCornerShape(8.dp))
+            )
+        }
+    }
+}
+
+@Composable
+fun Top20Section(programs: List<TvProgram>) {
```

Comment:
Top20Section은 NowBoarding을 하나로 묶어서 만들 수 있을 것 같습니다.
composable을 함수의 인자로 받을 수 있기 때문에 필요한 컴포저블은 따로 넣어주면 됩니다.
```suggestion
fun Top20Section(
programs: List<TvProgram>,
content: @Composable () -> Unit
) {
```